lib/ret_web/controllers/api/v1/media_search_controller.ex

```diff
@@ -29,25 +29,16 @@ defmodule RetWeb.Api.V1.MediaSearchController do
     conn |> render("index.json", results: results)
   end
 
-  def index(conn, %{"source" => "avatar_listings", "filter" => filter} = params) do
```

Comment:
If there was a reason we had these all being handled individually let me know, but it seems better handled at the `MediaSearchQuery` level

Reply:
no, probably just a spot we missed when refactoring